README.md

```diff
@@ -1,2 +1,171 @@
 # DianUserMigration-Java
-Code for mapping user data from periodic Happy Medium export to Bridge, for DIAN project
+This repository contains code for mapping user data from periodic Happy Medium (HM) exports to Synapse into a Sage Bridge project.
+
+# Project setup:
+
+Before you can run the code, your environment must have the following environmental variables set first:
+
+## Required
+**BR_EMAIL** - Your email to sign into the Arc Validation or DIAN bridge project
+**BR_PW** - Your password to sign into the Arc Validation or DIAN bridge project
+**BR_ID** - The bridge project ID for either the Arc Validation or DIAN bridge project
+
+**SYN_PAT** - Your Synapse access token to your account that has read/download access for the dian uat or dian prod Synapse projects.
+**SYN_PROJ_ID** - The Synapse Project ID for dian uat or dian prod.
+
+# Building the code
```

Comment:
This is definitely valuable info that I should add.  I'm releasing a new package on branch release/test2, and I'll update the ReadMe after I get the docker info I need.  There should be no args or entry point besides the environmental variables.

The _when_ is a good question.  I think that is something we will need to coordinate with HM.  I'll sync with Ann on this and get back to you on adding this to the ReadMe. For now, I'll say an hour after HM's data dump. 

Also, yes, the code is idempotent, even if you ran it twice at the same time, it would be fine, just inefficient.